docs/api/orders/order-fulfillment-statuses.md

```diff
@@ -0,0 +1,219 @@
+                "merchantOrderId": "5284911117",
+                "fulfillmentStatus": "sent to merchant",
+                "paymentStatus": "paid",
+                "quantity": 1
+            }
+        ],
+        "customer": {
+            "firstName": "Archimedes",
+            "lastName": "Syracuse",
+            "email": "gmail@gmail.com"
+        }
+    },
+    "errors": []
+}
+  ```
+
+  </div>
+</details>
+
+3. **fulfilled**: Once the merchant has sent the products to the customer.
+<details>
+  <summary>fulfilled order status response example</summary>
+  <div>
+
+  ```json
+  {
+    "data": {
+        "id": "31415926535897932384626433",
+        "orderNumber": 832795,
+        "externalId": "028841971693993751058209749445923",
+        "fulfillmentStatus": "fulfilled",
+        "shippingTracking": [
+            {
+                "carrier": "Shipping Carrier name",
+                "trackingNumber": "0348253421170679",
+                "trackingUrl": "https://trackingurl.com",
+                "products": [
+                    {
+                        "id": "078164062862089986280348253421170",
+                        "variantId": "679821480865132823066470938446095",
+                        "quantity": 1
+                    }
+                ]
+            }
+        ],
+        "products": [
+            {
+                "id": "078164062862089986280348253421170",
+                "variantId": "679821480865132823066470938446095",
+                "merchantOrderId": "5284911117",
+                "fulfillmentStatus": "fulfilled",
+                "paymentStatus": "paid",
+                "quantity": 1
+            }
+        ],
+        "customer": {
+            "firstName": "Archimedes",
+            "lastName": "Syracuse",
+            "email": "gmail@gmail.com"
+        }
+    },
+    "errors": []
+}
+  ```
+
+  </div>
+</details>
+
+This standard flow represents an order from submission to Bonsai till products being sent to the customer.
+
+---
+
+## PAYMENT SUCCESSFUL
+
+- **Description:** Payment for the order was successful.
+
+**Example:**
+
+An order that was submitted successfully to Bonsai.
+
+---
+
+## SENT TO MERCHANT
+
+- **Description:** This status indicates that the order information has been sent to the merchant for further processing.
+
+**Example:**
+
+An order that was submitted successfully to Bonsai, was submitted to a merchant, and is awaiting shipping details.
+
+---
+
+## PARTIALLY FULFILLED
+
+- **Description:** Order that has had product shipping information added to it for some of the products.
+
+**Example:**
+
+An order that was submitted successfully to Bonsai, was submitted to a merchant, and then has had some products shipped to the customer. This will happen when an order has products for multiple merchants as one merchant will ship products faster than the other. This is temporary until the shipping details for the other products are added.
```

Comment:
If we will list out all the statuses, let's include the exact format of such orders as well? Suppose it would be `Partially Fulfilled` which is inconsistent with the other statuses atm

Reply:
@AndreiBoris thanks for the catch here. I adjusted all the statuses so they have proper capitalization. Did you mean to add an example JSON for each order?